tutor/test_new_t1_14_TeacherReading.py

```diff
@@ -0,0 +1,2052 @@
+    @pytest.mark.skipif(str(132520) not in TESTS, reason='Excluded')
+    def test_teacher_save_a_draft_reading_for_individual_sections_132520(self):
+        """
+        Click on the 'Add Assignment' button
+        Click on the 'Add Reading' option
+        ***The teacher is taken to the Add Reading Assignment page*** (T1.14.001)
+
+        Enter an assignment name into the Assignment name text box
+        Click the 'Individual Sections' radio button
+        For each period reading is being assigned to:
+        Click on the Open Date text field and click on desired open date on calendar pop-up
+        Click on the Due Date text field and click on desired due date in the future on calendar pop-up
+        Click on the "+ Add Readings" button
+        Click on section(s) to add to assignment
+        Scroll to bottom of the page
+        Click on the "Add Readings" button
+        ***Takes user back to "Add reading" page *** (T1.14.004)
+        Click on the "Save As Draft" button (T1.14.005)
+
+        #Expected result
+        Takes user back to calendar dashboard. Assignment appears on user calendar dashboard on due date
+        with correct readings. 'draft' should appear before the assignment name.
+
+        """
+        # self.ps.test_updates['name'] = 't1.14.004' \
+        #                                + inspect.currentframe().f_code.co_name[4:]
+        # self.ps.test_updates['tags'] = ['t1', 't1.14', 't1.14.004', '7995']
+        self.ps.test_updates['passed'] = False
+
+        # Test steps and verification assertions
+        assignment_name = 'reading_002_%d' % (randint(100, 999))
+        assignment = Assignment()
+
+        # Open Add Reading page
+        self.teacher.assign.open_assignment_menu(self.teacher.driver)
+        self.teacher.find(By.LINK_TEXT, 'Add Reading').click()
+        assert ('reading/new' in self.teacher.current_url()), \
+            'not at add readings screen'
+
+        # Find and fill in title
+        self.teacher.wait.until(
+            expect.element_to_be_clickable(
+                (By.ID, 'reading-title')
+            )
+        ).send_keys(assignment_name)
+
+        # Set date
+        today = datetime.date.today()
+        start = randint(0, 6)
+        end = start + randint(1, 5)
+        opens_on = (today + datetime.timedelta(days=start)) \
+            .strftime('%m/%d/%Y')
+        closes_on = (today + datetime.timedelta(days=end)) \
+            .strftime('%m/%d/%Y')
+
+        # Find all individual periods
+        self.teacher.find(By.ID, 'show-periods-radio').click()
+        period_boxes = self.teacher.driver.find_elements(
+            By.XPATH, '//input[contains(@id, "period-toggle-period")]'
+        )
+        period_assignment = {}
+        for period in period_boxes:
+            period_assignment[
+                self.teacher.driver.find_element(
+                    By.XPATH, '//label[contains(@for, "%s")]' %period.get_attribute('id')).text
```

Comment:
make sure to watch for spacing...there should be a space between the mod and the variable for the string replacement ")]' % period..."